docs/content/schemas.md

```diff
@@ -450,6 +450,48 @@ When we derive a type from a schema, we try to match what is known and unknown i
 
 When overriding existing types, the dynamicity of the overridden prefix is preserved.
 
+### Supporting JSON Schema composition keywords 
+
+JSON Schema provides keywords such as `anyOf` and `allOf` to structure a complex schema. For `anyOf`, at least one subschemas must be true, and for `allOf`, all subschemas must be true. The type checker is able to identify such keywords and derive a more robust Rego type through more complex schemas. 
+
+#### `anyOf`
+
+Specifically, `anyOf` acts as an Rego Or type where at least one (can be more than one) of the subschemas is true. Consider the following Rego and schema file containing `anyOf`:
+
+`policy-anyOf.rego`
+```
+package kubernetes.admission  
+
+# METADATA
+# scope: rule
+# schemas: 
+#   - input: schema["input-anyOf"] 
+deny {                                                                
+  input.request.servers.versions == "Pod"                       
+}
+```
+
+`input-anyOf.json`: https://github.com/aavarghese/opa-schema-examples/blob/main/kubernetes/schemas/input-anyOf.json
```

Comment:
Hmm it is a pretty big file, but not terribly big. What do you think, would it be possible to cut it down a bit (removing descriptions, maybe?) so that it could fit in here? Having to open another window to understand the docs doesn't seem ideal 🤔 

Alternatively, we could use a `<details> ... </details>` section, perhaps? We've used some in the envoy docs...

Reply:
I deleted content that seems unnecessary to explain this example, hopefully the schema isn't too long here. What do you think? 

See 651e9803e56d4937c0e1a1b2afcc6d0aa4c6342d